src/models/rule.rs

```diff
@@ -51,6 +51,12 @@ pub struct Rule {
   #[pyo3(get)]
   query: TSQuery,
   /// The tag corresponding to the node to be replaced
```

Comment:
This doc is incorrect (duplicated from field `replace_node` below). Please change to a proper doc for this new `replace_idx` field :)